expense-frontend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20 as build
+
+WORKDIR /App
+
+COPY package*.json tsconfig.json ./
+
+RUN NODE_ENV=development npm i
+
+COPY . .
+
+
+RUN npm run build
```

Comment:
Good. After the build, what are you keeping?

Reply:
Did someone tried building this Dockerfile locally and running it?

Reply:
What about this ?